packages/ui/feature-builder-store/src/lib/store/builder/builder.selector.ts

```diff
@@ -194,6 +201,34 @@ const selectStepSelectedSampleData = createSelector(
     return undefined;
   }
 );
+const selectStepTestSampleData = createSelector(
+  selectCurrentStepSettings,
+  (settings) => {
+    if (settings) {
+      if (Object.keys(settings).find((s) => s === 'inputUiInfo')) {
+        const sampleDataSettings = (settings as Record<string, unknown>)[
+          'inputUiInfo'
+        ] as SampleDataSettings;
+        return sampleDataSettings.currentSelectedData;
+      }
+    }
+    return undefined;
+  }
+);
+const selectLastTestDate = createSelector(
+  selectCurrentStepSettings,
+  (settings) => {
+    if (settings) {
+      if (Object.keys(settings).find((s) => s === 'inputUiInfo')) {
```

Comment:
Use Pick, not sure if it works in array, this is hard coded and will cause an bug without an compilation error.


https://hackernoon.com/how-the-typescript-pick-type-works